forms-flow-api/src/formsflow_api/models/authorization.py

```diff
@@ -21,6 +21,7 @@ class AuthType(Enum):
     FORM = "FORM"
     FILTER = "FILTER"
     DESIGNER = "DESIGNER"
+    APPLICATION = "APPLICATION"
```

Comment:
the better way to use these variables we can create an enumerator class ?

Reply:
It is an enumeration class.
![image](https://github.com/AOT-Technologies/forms-flow-ai/assets/99173163/aab3d4a0-1c5e-4a07-9ac7-1e5e6ef13318)